docs/book/src/blockchain-development/abi_encoding.md

```diff
@@ -0,0 +1,17 @@
+# ABI Encoding
+
+Application binary interface (ABI) encoding typically enables programs to communicate with each other with the same data encoding system.
+
+The Sway language provides helpful traits and utilities to help with [Fuel ABI Encoding](https://docs.fuel.network/docs/specs/abi/) within the language, which is used across Sway programs.
+
+Sway, at its core, is agnostic to ABI encoding but preferences the Fuel ABI Encoding format.
```

Comment:
Would be nice to link to "Fuel ABI Encoding format". Because today what is at https://docs.fuel.network/docs/specs/abi/fn-selector-encoding/ needs to be updated. Maybe something like "for more details what abi_encode generates see https://docs.fuel.network/docs/specs/abi/argument-encoding/".

Reply:
Also useful to say that we are currently using "Version 1".

Reply:
Added, @xunilrj please see again.